src/cljs/chronograph_web/pages/create_organization/views.cljs

```diff
@@ -1,7 +1,7 @@
 (ns chronograph-web.pages.create-organization.views
   (:require [re-frame.core :as rf]
-            [chronograph-web.pages.create-organization.events :as events]
-            [chronograph-web.subscriptions :as subs]
+            [chronograph-web.pages.create-organization.events :as create-events]
```

Comment:
I'd used events since it made sense in context of this file. aliasing to create-* may not scale since there will probably be other create- pages.

Reply:
Done